src/sqlancer/yugabyte/ysql/YSQLSchema.java

```diff
@@ -160,6 +160,18 @@ protected static List<YSQLColumn> getTableColumns(SQLConnection con, String tabl
         return columns;
     }
 
+    public boolean getDatabaseIsColocated(SQLConnection con) {
+        try (Statement s = con.createStatement(); ResultSet rs = s.executeQuery("SELECT yb_is_database_colocated();")) {
+            rs.next();
+            String result = rs.getString(1);
+            // The query will result in a 'f' for a non-colocated database
+            return !"f".equals(result);
+
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
```

Comment:
Typically, we throw an `AssertionError`, but I think this should be fine as well